pkg/ratelimit/runner_test.go

```diff
@@ -82,7 +82,7 @@ func TestConcurrentRunner(t *testing.T) {
 		for i := 1; i < 11; i++ {
 			regionID := uint64(i)
 			if i == 10 {
-				regionID = 4
+				regionID = 6
```

Comment:
why change it？

Reply:
To pass this ut. Capacity changed the pendingTask index. To ensure the task is not consumed as much as possible, I have increased the region ID a bit more.